### ISSUE TITLE
[🐛 Bug]: no lines or helpful info displayed for "element not interactable"

### ISSUE THREAD

**Issue by zhangster** [webdriverio/webdriverio#8507]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

7.19

### Node.js Version

18.3.0

### Mode

Standalone Mode

### Which capabilities are you using?

```typescript
{
browserName: "chrome",
browserVersion: "latest",
platformName: "MacOS Catalina"
}
```

### What happened?

I'm getting this error.  It has no helpful info whatsoever, no line numbers, nothing about what object is not interactable.  below is the entirety of the output:

### How to reproduce the bug.

run npx wdio

### Relevant log output

```typescript
[chrome 103.0.5060.114 mac os x #0-0] element not interactable
  (Session info: chrome=103.0.5060.114)
[chrome 103.0.5060.114 mac os x #0-0] element not interactable: element not interactable
[chrome 103.0.5060.114 mac os x #0-0]   (Session info: chrome=103.0.5060.114)
[chrome 103.0.5060.114 mac os x #0-0]     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)


Spec Files:	 0 passed, 1 failed, 1 total (100% completed) in 00:01:31
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by christian-bromann** [comment]
Thanks for reporting!

I wonder if we loose the stacktrace when [parsing out the error](https://github.com/webdriverio/webdriverio/blob/21eb204333d2c66159e9b68325dfcf7afa7067c9/packages/webdriver/src/utils.ts#L213-L233) from the webdriver response.

Any contributions that resolves the bug are highly appreciated. Please take a look into our [contribution guidelines](https://webdriver.io/docs/contribute) and let us know if you have any questions. Cheers!

**Comment by praveendvd** [comment]
@christian-bromann this was the same behavior i was getting but was not able to reproduce in any other OS worked after updating wdio . @zhangster Please make sure you are using webdriverio latest , and please provide a reproducible example

**Comment by djv1001** [comment]
I'm currently seeing this on 7.23.0, on macOS 12.5, node 16.6.

Should I upgrade node?

**Comment by christian-bromann** [comment]
I pushed [an initiative](https://github.com/w3c/webdriver/issues/1679) to get more information from the protocol on the element that can not be interacted with. That said, we already have and can create the data needed to already provide more information today.

If anyone likes to take a stab at this I am happy to provide guidance. Currently I am personally focused on the `v8` release so this is less priority for me.

**Comment by prachiMuthal** [comment]
I'm currently seeing this on 7.24.0, on macOS 12.5, node 19.0.0

what should I do? any suggestions/help?

**Comment by prachiMuthal** [comment]
I am getting the same error and I'm currently seeing this on 7.24.0, on macOS 12.5, node 19.0.0

what should I do? any suggestions/help?

**Comment by djv1001** [comment]
@prachiMuthal I'm using TypeScript and just changed the compiler from ES6 to ES2019. 

I'm getting line numbers in the "element not... x" errors.

**Comment by margotvalencia23** [comment]
I'm having a similar issue (not only for "element not interactable") after updating webdriverIO to the latest version.

<img width="826" alt="image" src="https://user-images.githubusercontent.com/34656191/207952341-6cca4336-eaff-4d42-a5e4-d293ec4af2a9.png">

Basically, failed expectations show the stacktrace while any other kind of error does not show additional information. I didn't have this problem before the update.

**Comment by christian-bromann** [comment]
We greatly appreciate any contributions that help resolve the bug. While we understand that active contributors have their own priorities, we kindly request your assistance if you rely on this bug being fixed. We encourage you to take a look at our [contribution guidelines](https://webdriver.io/docs/contribute) or join our friendly [Discord](http://discord.webdriver.io/) development server, where you can ask any questions you may have. Thank you for your support, and cheers!

**Comment by cattermo** [comment]
If you get this error when using typescript, the error was fixed for me after I added `"target": "ES2019",` to the `tsconfig.json`. 

Thanks @djv1001  for the hint.

**Comment by Prabhs2** [comment]
@christian-bromann I have been facing this issue since I upgraded the version of WDIO v8 from WDIO 7. 

The exact version of WDIO I am currently using is 8.16.11. 

It seems as though it's happening due to the screen resolution/window size; however, when I tried with different window sizes and by setting the zoom to 100%, the scripts were still failing. 

It would be great if we could get a generic workaround to fix the issue otherwise, it's tedious to adjust the scripts everywhere to make it work.
